test/generate_property_json.js

```diff
@@ -157,42 +180,60 @@ function init (ops) {
   ops.push({ op: 'mkdir', path: '../outside' })
 }
 
-function start (ops) {
+function startWatcher (ops) {
   running = true
-  ops.push({ op: 'start' })
+  ops.push({ op: 'start_watcher' })
+}
+
+function startClient (ops) {
+  ops.push(sleep())
+  ops.push({ op: 'start_client' })
 }
 
-function run (ops) {
+function run (ops, availableOps) {
   const n = faker.random.number(nbRunOps)
   for (let i = 0; i < n; i++) {
-    const op = freq([
-      [5, createNewDir],
-      [3, createNewFile],
-      [1, recreateDeletedDir],
-      [1, recreateDeletedFile],
-      [1, updateFile],
-      [2, mvToNewPath],
-      [3, mvToDeletedPath],
-      [1, mvToOutside],
-      [1, mvFromOutside],
-      [5, rm],
-      [3, addReference],
-      [1, stopOrRestart],
-      [2, sleep]
-    ])
+    const op = freq(availableOps)
     ops.push(op)
   }
-  if (!running) {
-    ops.push({ op: 'restart' })
-  }
 }
 
-function generate () {
+function generateLocalWatcher () {
   let ops = []
   init(ops)
-  start(ops)
-  run(ops)
+  startWatcher(ops)
+  run(ops, localOps)
+  if (!running) {
+    ops.push({ op: 'restart' })
+  }
   console.log(JSON.stringify(ops))
 }
 
-generate()
+function generateTwoClients () {
+  let result = { desktop: [], laptop: [] }
+  // Flow doesn't want of:
+  // for (let ops of Object.values(result)) {
```

Comment:
Which version of `flow` does not want this `Object.values({ desktop: [], laptop: [] }`?
I checked on the flowjs online editor: https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBjOA7AzgFzDgCMArMAXjAG9UwwBDAJyYEYAuMAbQF0AaOoxYAmTr1QBfdFDhMAFDACmhAJZEoYAPJlFGfADoAbgxgBXRbjklSAShs1UASElA

Reply:
You need to add some calls to Array methods: https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBjOA7AzgFzDgCMArMAXjAG9UwwBDAJyYEYAuMAbQF0AaOoxYAmTr1QBfdFDhMAFDACmhAJZEoYAPJlFGfADoAbgxgBXRbjklSAShs1UASBXYV+OSpuTpp7HpU4YC5uVgAOuPa09Fh4hPhw8ZRgwoJw4fqhprgAFnLx8V4SQA

Reply:
My comment wasn't clear: you can use `for (let ops of Object.values(result)) {`, but `ops` is seen `mixed` and not as an array by flow.